src/optimize/BaseOptimizer.js

```diff
@@ -133,12 +133,20 @@ class BaseOptimizer {
       resolve: {
         extensions: ['.js', '.json', '.jsx', '.less', ''],
         postfixes: [''],
-        modulesDirectories: ['webpackShims', 'node_modules'],
+        modulesDirectories: ['webpackShims', 'node_modules', fromRoot('webpackShims'), fromRoot('node_modules')],
         loaderPostfixes: ['-loader', ''],
         root: fromRoot('.'),
         alias: this.env.aliases,
         unsafeCache: this.unsafeCache,
       },
+
+      resolveLoader: {
```

Comment:
it's just a part of webpack config http://webpack.github.io/docs/configuration.html#resolveloader